packages/components/src/utils/readonlyEventHandlers.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  type ChangeEvent,
+  type ChangeEventHandler,
+  type KeyboardEvent,
+  type KeyboardEventHandler,
+  type MouseEvent,
+  type MouseEventHandler,
+} from 'react';
+
+// Handlers ved readonly, da det ikke støttes ut av boksen for enkelte inputelementer (select, radio, checkbox)
+
+export const readOnlyEventHandlers = <T,>(
+  readOnly?: boolean,
+  onMouseDown?: MouseEventHandler<T>,
+  onKeyDown?: KeyboardEventHandler<T>,
+  preventAllKeys?: boolean,
+  keys: Array<string> = [],
+) => {
+  const handlers = {
+    onMouseDown: (evt: MouseEvent<T>) => {
```

Comment:
Denne kan kanskje bare bruke samme implementasjon som `readonlyMouseDownHandler`
```suggestion
    onMouseDown: readOnlyMouseDownHandler(readOnly, onMouseDown),
```